ra-optimisation/sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/RangeActionSensitivitiesProvider.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.sensitivity_computation;
+
+import com.farao_community.farao.commons.FaraoException;
+import com.farao_community.farao.data.crac_api.Cnec;
+import com.farao_community.farao.data.crac_api.Crac;
+import com.farao_community.farao.data.crac_api.NetworkElement;
+import com.farao_community.farao.data.crac_api.RangeAction;
+import com.powsybl.iidm.network.*;
+import com.powsybl.sensitivity.SensitivityFactor;
+import com.powsybl.sensitivity.SensitivityFunction;
+import com.powsybl.sensitivity.SensitivityVariable;
+import com.powsybl.sensitivity.factors.BranchFlowPerInjectionIncrease;
+import com.powsybl.sensitivity.factors.BranchFlowPerPSTAngle;
+import com.powsybl.sensitivity.factors.BranchIntensityPerPSTAngle;
+import com.powsybl.sensitivity.factors.functions.BranchFlow;
+import com.powsybl.sensitivity.factors.functions.BranchIntensity;
+import com.powsybl.sensitivity.factors.variables.InjectionIncrease;
+import com.powsybl.sensitivity.factors.variables.PhaseTapChangerAngle;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * @author Philippe Edwards {@literal <philippe.edwards at rte-france.com>}
+ */
+public class RangeActionSensitivitiesProvider extends AbstractSimpleSensitivityProvider {
```

Comment:
Only this one is SensitivitiesProvider when the other ones are SensitivityProvider. Maybe you're right and sensitivities - plural - makes more sense but it has to be consistent

Reply:
changed all to singular